charts/argo-workflows/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+1. Get Argo Server external IP/domain by running:
+
+kubectl --namespace {{ .Release.Namespace }} get services -o wide | grep {{ .Release.Name }}-{{ .Values.server.name }}
+
+2. Submit the hello-world workflow by running:
+
+argo submit https://raw.githubusercontent.com/argoproj/argo/master/examples/hello-world.yaml --watch
```

Comment:
This link should be updated to https://raw.githubusercontent.com/argoproj/argo-workflows/master/examples/hello-world.yaml